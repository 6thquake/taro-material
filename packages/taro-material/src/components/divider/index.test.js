import Nerv from 'nervjs'
import { renderToString } from 'nerv-server'

import AtDivider from '../../../.temp/components/divider/index'

describe('AtDivider Snap', () => {
  it('render initial AtDivider', () => {
    const component = renderToString(<AtDivider isTest />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props customStyle', () => {
    const component = renderToString(<AtDivider isTest customStyle='color:red;' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props className', () => {
    const component = renderToString(<AtDivider isTest className='test' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props content', () => {
    const component = renderToString(<AtDivider isTest content='content' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props height', () => {
    const component = renderToString(<AtDivider isTest height='120' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props fontColor', () => {
    const component = renderToString(<AtDivider isTest fontColor='#fff' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props fontSize', () => {
    const component = renderToString(<AtDivider isTest fontSize='56' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props lineColor', () => {
    const component = renderToString(<AtDivider isTest lineColor='#fff' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtDivider -- props childen', () => {
    const component = renderToString(<AtDivider isTest>test</AtDivider>)
    expect(component).toMatchSnapshot()
  })
})
