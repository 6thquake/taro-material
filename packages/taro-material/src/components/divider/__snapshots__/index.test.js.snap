// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtDivider Snap render AtDivider -- props childen 1`] = `"<div style=\\"height:2.3894rem;\\" class=\\"at-divider\\"><div style=\\"color:#6190E8;font-size:0.6827rem;\\" class=\\"at-divider__content\\">test</div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render AtDivider -- props className 1`] = `"<div style=\\"height:2.3894rem;\\" class=\\"at-divider test\\"><div style=\\"color:#6190E8;font-size:0.6827rem;\\" class=\\"at-divider__content\\"></div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render AtDivider -- props content 1`] = `"<div style=\\"height:2.3894rem;\\" class=\\"at-divider\\"><div style=\\"color:#6190E8;font-size:0.6827rem;\\" class=\\"at-divider__content\\">content</div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render AtDivider -- props customStyle 1`] = `"<div style=\\"height:2.3894rem;color:red;\\" class=\\"at-divider\\"><div style=\\"color:#6190E8;font-size:0.6827rem;\\" class=\\"at-divider__content\\"></div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render AtDivider -- props fontColor 1`] = `"<div style=\\"height:2.3894rem;\\" class=\\"at-divider\\"><div style=\\"color:#fff;font-size:0.6827rem;\\" class=\\"at-divider__content\\"></div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render AtDivider -- props fontSize 1`] = `"<div style=\\"height:2.3894rem;\\" class=\\"at-divider\\"><div style=\\"color:#6190E8;font-size:1.1947rem;\\" class=\\"at-divider__content\\"></div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render AtDivider -- props height 1`] = `"<div style=\\"height:2.56rem;\\" class=\\"at-divider\\"><div style=\\"color:#6190E8;font-size:0.6827rem;\\" class=\\"at-divider__content\\"></div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render AtDivider -- props lineColor 1`] = `"<div style=\\"height:2.3894rem;\\" class=\\"at-divider\\"><div style=\\"color:#6190E8;font-size:0.6827rem;\\" class=\\"at-divider__content\\"></div><div style=\\"background-color:#fff;\\" class=\\"at-divider__line\\"></div></div>"`;

exports[`AtDivider Snap render initial AtDivider 1`] = `"<div style=\\"height:2.3894rem;\\" class=\\"at-divider\\"><div style=\\"color:#6190E8;font-size:0.6827rem;\\" class=\\"at-divider__content\\"></div><div style=\\"background-color:#CCC;\\" class=\\"at-divider__line\\"></div></div>"`;
